tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Models/Twitter/WebhookDMResultTests.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Bot.Builder.Community.Adapters.Twitter.Webhooks.Models.Twitter;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Bot.Builder.Community.Adapters.Twitter.Webhooks.Models;
+
+namespace Bot.Builder.Community.Adapters.Twitter.Tests.Webhooks.Models.Twitter
+{
+    [TestClass]
+    [TestCategory("Twitter")]
+    public class WebhookDMResultTests
+    {
+        [TestMethod]
+        public void WebhookDMResultPropertiesShouldBeSetSuccessfully()
+        {
+            var webhookDmResult = new WebhookDMResult()
+            {
+                Users = new Dictionary<string, TwitterUser>(),
+                Events = new List<DMEvent>()
+            };
+
+            var jsonResult = JsonConvert.DeserializeObject<WebhookDMResult>(JsonConvert.SerializeObject(webhookDmResult));
+
+            Assert.AreEqual(webhookDmResult.Users.GetType(), jsonResult.Users.GetType());
+            Assert.AreEqual(webhookDmResult.Events.GetType(), jsonResult.Events.GetType());
+        }
+
+        [TestMethod]
+        public void WebhookDMResultShouldConvertoToDirectMessageEventSuccessfully()
+        {
+            DirectMessageEvent dmEvent = new WebhookDMResult()
```

Comment:
These are redundant. Please remove them all.
```suggestion
            DirectMessageEvent dmEvent = new WebhookDMResult
```